cmd/datamon/cmd/bundle_get.go

```diff
@@ -65,7 +70,8 @@ exits with ENOENT status otherwise.`,
 		log.Println(buf.String())
 	},
 	PreRun: func(cmd *cobra.Command, args []string) {
-		config.populateRemoteConfig(&datamonFlags)
+		optionInputs := newCliOptionInputs(config, &datamonFlags)
+		optionInputs.populateRemoteConfig()
```

Comment:
more succinct, actionable comment than [this](https://github.com/oneconcern/datamon/pull/379#discussion_r374950405) is that, since i'm not grokking what the uppercased private function would mean, exactly – what's an example of a programming language that includes the paradigm/concept that this convention is mimicing? – i'd prefer to 

1. land the patch without changing the `cmd` pkg api
2. click approve on a patch wherein appropriate functions are uppercased (quick Rename in any ide) and we have a notes on github about the meaning of these uppercased functions

Reply:
will make the change to include the `With*()` method.

Reply:
after following [the suggestion](https://github.com/oneconcern/datamon/pull/379#discussion_r374967426) to return errors from `cliOptionInputs` methods rather than (indirectly) `os.Exit`ing, `populateRemoteConfig` will return errors and this idiom won't apply.

Reply:
yes indeed